pkg/api-server/inspect_endpoints_test.go

```diff
@@ -170,7 +170,8 @@ func (b *dataplaneBuilder) outbound8080(service, ip string) *dataplaneBuilder {
 	return b
 }
 
-func (b *dataplaneBuilder) admin(port uint32) *dataplaneBuilder {
+func (b *dataplaneBuilder) admin() *dataplaneBuilder {
+	var port uint32 = 3301
```

Comment:
why do we want to make the port hardcoded and not configurable?

Reply:
It's something unparam complains about, it's always 3301. IMO it's less noisy if the port is unimportant (which it is, since it's always 3301)